packages/components/src/Link/docs/Link.stories.tsx

```diff
@@ -0,0 +1,246 @@
+import { SparklesIcon } from "@hopper-ui/icons";
+import { Div } from "@hopper-ui/styled-system";
+import type { Meta, StoryObj } from "@storybook/react";
+import { createMemoryRouter, RouterProvider, useNavigate } from "react-router-dom";
+
+import { HopperProvider } from "../../HopperProvider/index.ts";
+import { Stack } from "../../layout/index.ts";
+import { Text } from "../../Text/index.ts";
+import { Link } from "../src/Link.tsx";
+
+import { FrogImage } from "./assets/index.ts";
+
+/**
+ * Links allow users to navigate to a different location. They can be presented inline inside a paragraph or as standalone text.
+ *
+ * [View repository](https://github.com/gsoft-inc/wl-hopper/tree/main/packages/components/src/Link/src)
+ * -
+ * [View package](https://www.npmjs.com/package/@hopper-ui/components)
+ * -
+ * View storybook TODO
+ */
+const meta = {
+    title: "Docs/Link",
+    tags: ["autodocs"],
+    parameters: {
+        // Disables Chromatic's snapshotting on documentation stories
+        chromatic: { disableSnapshot: true }
+    },
+    component: Link,
+    args: {
+        children: "Learn more",
+        href: "#"
+    }
+} satisfies Meta<typeof Link>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+/**
+ * A default link
+ */
+export const Default: Story = {
+};
+
+/**
+ * A link can use a different style.
+ */
+export const Variants : Story = {
+    render: args => (
+        <>
+            <Link variant="primary" {...args}>Primary</Link>
+            <Link variant="secondary" {...args}>Secondary</Link>
+        </>
+    )
+};
+
+/**
+ * All links can have a quiet style, without an underline. This style should only be used when the placement and context of the link is explicit enough that a visible underline isn’t necessary.
+ */
+export const Quiet: Story = {
+    render: args => (
+        <p>Would you like to <Link {...args}>learn more</Link> about this fine component?</p>
+    ),
+    args:{
+        isQuiet: true
+    }
+};
+
+/**
+ * A text link can contain icons
+ */
+export const Icon: Story = {
+    render: args => (
+        <>
+            <Link variant="primary" {...args} />
+            <Link variant="secondary" {...args} />
+        </>
+    ),
+    args: {
+        children: [
+            <Text key="1">Learn more</Text>,
+            <SparklesIcon key="2" />
+        ]
+    }
+};
+
+/**
+ * **Non standard** start icons can be provided to handle special cases. However, think twice before adding *start* icons, *end* icons should be your go to.
```

Comment:
"However, think twice before adding *start* icons"
does the above sound a little threatening? Maybe not, just made me giggle a little.